collect_app/src/main/java/org/odk/collect/android/preferences/PreferenceKeys.java

```diff
@@ -1,57 +1,57 @@
 package org.odk.collect.android.preferences;
 
 public final class PreferenceKeys {
-    public static final String KEY_LAST_VERSION = "lastVersion";
-    public static final String KEY_FIRST_RUN = "firstRun";
-    public static final String KEY_SHOW_SPLASH = "showSplash";
-    public static final String KEY_SPLASH_PATH = "splashPath";
-    public static final String KEY_FONT_SIZE = "font_size";
-    public static final String KEY_DELETE_AFTER_SEND = "delete_send";
-    public static final String KEY_ANALYTICS = "analytics";
-
-    public static final String KEY_PROTOCOL = "protocol";
-           static final String KEY_PROTOCOL_SETTINGS = "protocol_settings";
-
-    // leaving these in the main screen because username can be used as a
+    public static final String KEY_LAST_VERSION             = "lastVersion";
+    public static final String KEY_FIRST_RUN                = "firstRun";
+    public static final String KEY_SHOW_SPLASH              = "showSplash";
+    public static final String KEY_SPLASH_PATH              = "splashPath";
+    public static final String KEY_FONT_SIZE                = "font_size";
+    public static final String KEY_DELETE_AFTER_SEND        = "delete_send";
+    public static final String KEY_ANALYTICS                = "analytics";
+
+    public static final String KEY_PROTOCOL                 = "protocol";
+           static final String KEY_PROTOCOL_SETTINGS        = "protocol_settings";
+
+    // leaving these in the main screen because username can be used as adminKey
```

Comment:
Fixed, and pushed, in 303-activities-checkable-lists branch (the other pull request).